k8s/upgrade-job/src/helm/upgrade.rs

```diff
@@ -134,18 +153,35 @@ impl HelmUpgradeBuilder {
         }
 
         // Validating upgrade path.
+        let mut invalid_upgrade_path: PathBuf;
+
+        match self.upgrade_path_dir {
+            Some(up_path) => invalid_upgrade_path = up_path,
+            None => {
+                return FilePathNotPresent.fail();
+            }
+        }
+        invalid_upgrade_path.push("unsupported_versions.yaml");
```

Comment:
```suggestion
        let invalid_upgrade_path = match self.upgrade_path_dir {
            Some(path) => Ok(path.join("unsupported_versions.yaml")),
            None => Err(Error::FilePathNotPresent),
        }?;
```
Also, should "unsupported_versions.yaml" be a const?

Reply:
passing the complete file path now.